Mail/MailerHelper.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\FormBundle\Mail;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Email;
+
+class MailerHelper implements HelperInterface
+{
+    /**
+     * @var MailerInterface
+     */
+    private $mailer;
+
+    /**
+     * @var null|string
+     */
+    protected $toMail;
+
+    /**
+     * @var null|string
+     */
+    protected $fromMail;
+
+    /**
+     * @var string|null
+     */
+    protected $sender;
+
+    /**
+     * @var LoggerInterface
+     */
+    protected $logger;
+
+    public function __construct(
+        MailerInterface $mailer,
+        ?string $fromMail,
+        ?string $toMail,
+        ?string $sender = null,
+        ?LoggerInterface $logger = null
+    ) {
+        $this->mailer = $mailer;
+        $this->toMail = $toMail;
+        $this->fromMail = $fromMail;
+        $this->sender = $sender;
+        $this->logger = $logger ?: new NullLogger();
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function sendMail(
+        $subject,
+        $body,
+        $toMail = null,
+        $fromMail = null,
+        bool $html = true,
+        $replyTo = null,
+        array $attachments = [],
+        $ccMail = [],
+        $bccMail = [],
+        $plainText = null
+    ): int
+    {
+        $message = new Email();
+
+        $this->setHeaders(
+            $message,
+            $subject ?? '',
+            $this->parseToAddresses($fromMail ?? $this->fromMail),
+            $this->parseToAddresses($toMail ?? $this->toMail),
+            $this->parseToAddresses($replyTo),
+            $this->parseToAddresses($ccMail),
+            $this->parseToAddresses($bccMail),
+            $this->sender ? new Address($this->sender) : null
+        );
+        $this->setBody($message, $html, $body, $plainText);
+        $this->setAttachments($message, $attachments);
+
+        $this->logMessage(
+            $fromMail ?? $this->fromMail,
+            $toMail ?? $this->toMail,
+            $replyTo,
+            $subject ?? '',
+            $ccMail,
+            $bccMail,
+            $plainText
+        );
+
+        $this->mailer->send($message);
+        return 0;
```

Comment:
Empty line should be here before the return.

```suggestion

        return 0;
```